neuralmonkey/config/disambiguate.py

```diff
@@ -0,0 +1,141 @@
+"""Module for disambiguating and enhancing configuration."""
```

Comment:
To není _disambiguate_, protože ta konfigurace není ambiguious. Podle mě je to spíš něco jako _resolving parameter values_.

Reply:
a jak bys pojmenoval ten modul? mě to jako disambiguace přijde, když tam můžu mít třeba val_dataset nebo trainers jako seznam i jako jednu hodnotu.

Reply:
Normalize nebo standardize

Reply:
done